doc/source/tutorials/emulator.rst

```diff
@@ -137,29 +148,42 @@ We are now going to give an example on how to setup the `platform.py` file.
     # emulator / platform.py
 
 
-    from qibolab import ConfigKinds, IqChannel, Hardware, Qubit
-    from qibolab.instruments.emulator import EmulatorController, HamiltonianConfig
+    from qibolab import ConfigKinds, DcChannel, IqChannel, Platform, Qubit, Hardware
+    from qibolab.instruments.emulator import (
+        DriveEmulatorConfig,
+        EmulatorController,
+        FluxEmulatorConfig,
+        HamiltonianConfig,
+    )
 
-    ConfigKinds.extend([HamiltonianConfig])
+    ConfigKinds.extend([HamiltonianConfig, DriveEmulatorConfig, FluxEmulatorConfig])
 
 
     def create() -> Hardware:
-        """Create an emulator hardware."""
-        qubits = {Qubit.default(q) for q in range(1)}
-        channels = {
-            qubit.drive: IqChannel(mixer=None, lo=None) for qubit in qubits.values()
-        }
+        """Create emulator platform with flux-tunable qubit."""
+        qubits = {}
+        channels = {}
+
+        for q in range(1):
+            qubits[q] = qubit = Qubit.default(q)
+            channels |= {
+                qubit.drive: IqChannel(mixer=None, lo=None),
+                qubit.flux: DcChannel(),
+            }
 
         # register the instruments
         instruments = {
-            "dummy": EmulatorController(address="0.0.0.0", channels=channels),
+            "emulator": EmulatorController(address="0.0.0.0", channels=channels),
         }
 
         return Hardware(
             instruments=instruments,
             qubits=qubits,
         )
 
+
 We can observe that we need to allocate an object of type ``EmulatorController`` where we load the channels.
-Note that in order to enables the config to support the Hamiltonian configuration we are adding it explicitly
-in the statement ``ConfigKinds.extend([HamiltonianConfig])``.
+Note that in order to enables the config to support the Hamiltonian configuration as well as drive and flux channels for the emualtor
```

Comment:
```suggestion
Note that in order to enable the config to support the Hamiltonian configuration as well as drive and flux channels for the emulator
```

Just as a disclaimer: I'm not really checking grammar consistently. And sometimes I'm not even reporting errors when I find them.
This to say that is better to rely on a spell checker than on my reviews :P